### PR TITLE
Update docs setup, remove virtualenv use in dev image, add VSCode debug support

### PR DESCRIPTION
**Pull request by davidlougheed**

### CODE REVIEW

File: docs/requirements.txt
Comment (v-rocheleau) [inline docs/requirements.txt:1]: We dont use poetry for the docs?

File: bento.dev.Dockerfile [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
+LABEL org.opencontainers.image.description="Local development image for Katsu."
+LABEL devcontainer.metadata='[{ \
+  "remoteUser": "bento_user", \
+  "customizations": { \
+    "vscode": { \
+      "extensions": ["ms-python.python", "eamodio.gitlens"], \
+      "settings": {"workspaceFolder": "/app"} \
+    } \
+  } \
+}]'
```
Comment (v-rocheleau) [inline bento.dev.Dockerfile:12]: Could we have a short section in the README explaining how to use this? Looks neat!

**Review by v-rocheleau — COMMENTED** [review]
Looks good, would just like to know more about the vscode thingy

File: docs/requirements.txt
Comment (davidlougheed) [reply docs/requirements.txt:1]: it's processed by readthedocs, I didnt feel like customizing their build process...

File: bento.dev.Dockerfile (hunk 1 above)
Comment (davidlougheed) [reply bento.dev.Dockerfile:12]: will do!

Comment (davidlougheed) [reply bento.dev.Dockerfile:12]: added

**Review by v-rocheleau — APPROVED** [review]

**Pull request merged**